modules/nextflow/src/main/groovy/nextflow/conda/CondaOptions.groovy

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2020-2021, Seqera Labs
+ * Copyright 2013-2019, Centre for Genomic Regulation (CRG)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package nextflow.conda
+
+import groovy.transform.CompileStatic
+import groovy.transform.EqualsAndHashCode
+import groovy.transform.PackageScope
+import groovy.transform.ToString
+
+/**
+ * Model Conda options.
+ *
+ * @author Abhinav Sharma <abhi18av@outlook.com>
+ */
+@CompileStatic
+@ToString(includeNames = true)
+@EqualsAndHashCode(includeFields = true)
+class CondaOptions {
+
+    private Boolean useMamba
```

Comment:
Perhaps, I should add the `Path path` option here as well?

Reply:
Instead of specifying a process level better to add this setting in the conga config (ie nextflow config file). see here for example 

https://github.com/nextflow-io/nextflow/blob/914d7cb40bf58ff977717cd137352e7dfe97dbdb/modules/nextflow/src/main/groovy/nextflow/conda/CondaCache.groovy#L82-L93

Reply:
Okay, so the user would only need to update the `nextflow.config` file like so 

```nextflow
conda {
   useMamba = true
}
```

Hmm, this is interesting and cleaner as it localizes the overall configs.

Cool, I'm implementing this one now.